$NetBSD: patch-mssqldbmodule_c,v 1.1.2.2 2011/10/15 09:42:36 tron Exp $

Fix build with latest freetds, which now provides BYTE and LPBYTE.

--- mssqldbmodule.c~	2006-09-22 19:44:17.000000000 +0000
+++ mssqldbmodule.c
@@ -67,9 +67,6 @@
   #define SQLVARCHAR	SYBVARCHAR
   #define SQLCHAR	SYBCHAR
   #define SQLTEXT	SYBTEXT
-
-  #define BYTE		unsigned char
-  typedef unsigned char *LPBYTE;
 #endif
 
 #define TYPE_STRING	1
